From a5293f12e3e0d7de50ee95c4bd4709a7cb660c06 Mon Sep 17 00:00:00 2001
From: Hai Pham <hai.pham.ud@renesas.com>
Date: Tue, 27 May 2025 05:25:55 +0700
Subject: [PATCH 09/46] pinctrl: renesas: Show bit position in config write

Show bit position in config write debug log, which is helpful for cases
where the p port setting is applied at the exact p bit position

Signed-off-by: Hai Pham <hai.pham.ud@renesas.com>
---
 drivers/pinctrl/renesas/core.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/pinctrl/renesas/core.c b/drivers/pinctrl/renesas/core.c
index ff27b9fea4ac..b73b6d88d347 100644
--- a/drivers/pinctrl/renesas/core.c
+++ b/drivers/pinctrl/renesas/core.c
@@ -229,8 +229,8 @@ static void sh_pfc_write_config_reg(struct sh_pfc *pfc,
 	sh_pfc_config_reg_helper(pfc, crp, field, &mapped_reg, &mask, &pos);
 
 	dev_dbg(pfc->dev, "write_reg addr = %x, value = 0x%x, field = %u, "
-		"r_width = %u, f_width = %u\n",
-		crp->reg, value, field, crp->reg_width, hweight32(mask));
+		"pos = %u, r_width = %u, f_width = %u\n",
+		crp->reg, value, field, pos, crp->reg_width, hweight32(mask));
 
 	mask = ~(mask << pos);
 	value = value << pos;
-- 
2.34.1

